content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -28,6 +28,27 @@ located above the project table.
 
 ![](https://static.iterative.ai/img/studio/view_components_1.gif)
 
+### Nested branches
+
+When a Git branch is merged into another branch (e.g., `main`), two
+possibilities exist:
+
+- The user continues to push more commits to the merged branch, which means that
+  the merged branch contains some commits that are not present in `main`. In
+  this scenario, the project table will display both the `main` branch and the
+  merged branch and the merged branch will show the commits that are not present
+  in `main`.
+
+- The user does NOT push any more commits to the merged branch, which means that
+  the merged branch does NOT contain any commits that are not present in `main`.
+
+  In this scenario, Iterative Studio considers the merged branch as **"nested"**
+  and does not display it as a separate branch in the project table. This helps
+  to keep the project table concise when there are many branches that get merged
+  into `main` (or some other branch) over time; including all the merged
+  branches in the project table would make the table very large without adding
+  much informational value.
```

Comment:
Wanted to suggest some rewording, my key goals were:
- not refering to the branch as "merged" if its tip is not merged.
- clarifying that this example can be projected on any case where some "trunk" branch accumulates changes from working/feature branches.
- using the terms "active" and "inactive" to illustrate the perceived importance of the branch to the user.

I ended up with that:

> 
> For illustration purposes, we'll assume a Git branch, named `my-exp`, which includes some unique commits, was merged into the `main` branch (pull request target). In this example `main` is the "trunk" in the team, where active work is being merged to. In reality, this can also any branch that the team chooses of course, like `master, `dev`, `develop`, or some integration/release branch. Often this is also the default branch for the repository.
> 
> After a PR from a Git branch is merged, two possibilities exist:
> 
> 1. `my-exp` branch is still active - the user continues to push commits to it after it was merged to `main` on some pull request commit, which is no longer its tip. Since the branch contains new unique commits, the project table will display both the target branch (`main`) and the feature branch (`my-exp`) separately. Here, `my-exp` will only show the new commits that are not part of `main`.
> 
> 2. `my-exp` is inactive - No new commits were pushed to `my-exp` after it was merged to `main`. In this scenario, Iterative Studio considers `my-exp` branch as **"nested branch"** and does not display it as a separate branch in the project table by default. This helps to keep the project table concise and reduce clutter that can accumulate over time when inactive branches are not cleaned from the project repository. Those "inactive branches" usually carry no new information for the purpose of managing experiments in Studio. If you would like to force display such an "inactive branch", use the `"Commits on branch"=my-exp` display filter.

wdyt @tapadipti ?
Feel free to adopt in whole/parts or drop

Reply:
Thanks Oded. I've adopted this partially (in [this commit](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744)), and I think it's much clearer now.